samples/cpp/cloning_gui.cpp

```diff
@@ -453,7 +453,7 @@ int main()
 
     for(;;)
     {
-        char key = (char) waitKey(0);
+        int key = waitKey(0) & 0xFF;
 
         if(key == 'd' && flag3 == 0)
```

Comment:
@StevenPuttemans , @alalek don't you think that after making `c` of type `int` this comparison can be `true` or `false` depending on the `signed` or `unsigned` `char` type default?
I would prefer leaving `char` type for `c`...